src/Cryptography/Sodium/Main.hs

```diff
@@ -0,0 +1,75 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE CApiFFI #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+-- | Module: Cryptography.Sodium.Main
+-- Description: Wrappers for initialization
+-- Copyright: (C) Koz Ross 2022
+-- License: BSD-3-Clause
+-- Maintainer: koz.ross@retro-freedom.nz
+-- Stability: Stable
+-- Portability: GHC only
+--
+-- @libsodium@ requires initialization before use. We provide a binding to the
+-- initialization function, as well as some high-level wrappers for applications
+-- to use without needing to call an FFI wrapper.
+--
+-- = Note
+--
+-- If you are using @cryptography-libsodium@ as a dependency for a library, you
+-- are probably not interested in this; it's designed for application authors who
+-- need capabilities provided by @cryptography-libsodium@.
+module Cryptography.Sodium.Main
+  ( -- * High-level wrappers
+    secureMain,
+    secureMainWith,
+
```

Comment:
extraneous empty line

Reply:
I can't remove this - `ormolu` forces it between headers in export lists.